lib/samson/secrets/hashicorp_vault_backend.rb

```diff
@@ -59,6 +75,10 @@ def vault_client
           @vault_client ||= VaultClient.new
         end
 
+        def wrap_vault_errors(message)
+          raise Samson::Secrets::BackendError.new("Vault backend is down!! " + message)
```

Comment:
maybe something more vague like "Error talking to vault backend: #{message}" so we don't claim something that is wrong and lead ops onto a wild goose chase